docs/conf.py

```diff
@@ -180,6 +180,7 @@ def ignore_down_providers():
     if resp.status_code != 200:
         return []
     body = BeautifulSoup(resp.text)
+    # pylint: disable=not-an-iterable
     nodes = list(str(item) for item in body.find_all("tr") if any(status in str(item) for status in ["DOWN", "UP"]))
```

Comment:
Can you change it to be multi-line and place the `# pylint: disable=not-an-iterable` on the line of the specific part so it is obvious which one is problematic?